comm/README.md

```diff
@@ -87,28 +88,32 @@ However it gets run, it responds to the following arguments:
 #### Bootstrapping
 
 By default, the node will not attempt to bootstrap into any other network and so will create a brand new network with
-only the one
+only the one node sitting there, waiting for other nodes to contact it. Giving the argument
 
 ```
---bootstrap rnode://ace40ebca0924eb797bb69dfda04f5d9@216.83.154.106:30012
+--bootstrap rnode://RChainAuthoritativeBootstrapNode@216.83.154.106:30012
```

Comment:
Is the 216.83.154.106 ip address likely to change?  Can we use DNS to give it a name off of pyrofex.net or rchain.coop?

Reply:
I don't know. Eventually we'll want to round-robin some dns entries to several servers, I imagine. Whether this one will change is a question for @jeremybusk.

Reply:
Definitely dns in round robin.  We just need to decide on domain. I'll check with Nash and see what domains we have and what board wants to use. Maybe, if it ever calls for it. 

https://easyipv6.wordpress.com/2013/06/28/building-a-fast-content-network-with-anycast/
You guys might find it at least interesting. It's what Everyone uses like Google DNS/YouTube/Netflix etc. Any CDN. The cool thing is you can actually use the same ips around the world. It does take a little setup.